doc/source/user_guide/io.rst

```diff
@@ -5240,6 +5240,7 @@ Write to a feather file.
 Read from a feather file.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
why is this added here?

Reply:
Ah, I suppose this was already merged before as https://github.com/pandas-dev/pandas/pull/40542